external/java8-tests/src/test/java/test/org/apache/spark/java8/Java8RDDAPISuite.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package org.apache.spark;
+package test.org.apache.spark.java8;
```

Comment:
the java8 part is to make it easier to run all java 8 tests. not that big of a deal though. the more important one is to make the beginning part consistent.